teaser/project.py

```diff
@@ -981,6 +991,11 @@ def export_ibpsa(self, library="AixLib", internal_id=None, path=None):
             "IDEAS",
         ], ass_error_1
 
+        if not self.buildings:
+            raise ValueError("The project includes no buildings to export.")
+        if internal_id is not None and not any(bldg.internal_id == internal_id for bldg in self.buildings):
+            raise ValueError(f"Building with internal_id {internal_id} not found in the project.")
+
```

Comment:
maybe add a util function for this check, as it is always the same? Else, looks good.